common/changes/@gooddata/sdk-ui-all/ch3_2023-09-06-13-17.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@gooddata/sdk-ui-all",
+      "comment": "To enhance performance, the attribute filter fetches the count of elements only for child filters, rather than fetching the total count of elements by default.",
```

Comment:
Optimized attribute filter to fetch element counts only for child filters, reducing overhead and improving performance by not fetching the total count of elements by default.